### PR TITLE
Time to readd 'Save actions for later'?

### PR DESCRIPTION
**Pull request by 7omas**
Refactor of changeEvent
The only three API calls that are currently not available in Ontime seems to be linkStart, endAction and timerType

Extended the support for more types of cssColor notations.
Color API does not allow none or empty string to select no color, workaround is to currently send a dark gray
A fix for this could perhaps be to add 'none'=''; to the cssColor array in Ontime

### CODE REVIEW

File: Ontime.js [hunk 1]
```diff
@@ -303,52 +309,47 @@ function changeEvent(
...
+    'note': ['note', note],
+    'cue': ['cue',cue],
+    'timeStart': ['timeStart',parseInt(timeStart)],
+    'linkStart': ['linkStart',linkStart],
+    'timeEnd': ['timeEnd',parseInt(timeEnd)],
+    'duration': ['duration',parseInt(duration)],
+    'skip': ['skip',skip],
+    'isPublic': ['isPublic',public],
+    'timerType': ['timerType',timerType], // Find correct values for function?
+    'endAction': ['endAction',endAction], // Find correct values for function?
```
Comment (alex-Arc) [inline Ontime.js]: > Find correct values for function?

What are you missing here?

**Comment by alex-Arc** [comment]
> A fix for this could perhaps be to add 'none'=''; to the cssColor array in Ontime

Good point I will look into it

File: Ontime.js (hunk 1 above)
Comment (7omas) [reply Ontime.js]: Running these as is, results in Error: Invalid value received

Comment (alex-Arc) [reply Ontime.js]: Oh, ok will look into that as well 
It looks correct from your end

**Review by alex-Arc — APPROVED** [review]

**Review by alex-Arc — APPROVED** [review]

**Pull request merged**